SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -386,6 +399,51 @@ TArray<Worker_InterestOverride> USpatialSender::CreateComponentInterest(AActor*
 	return ComponentInterest;
 }
 
+RPCsOnEntityCreation USpatialSender::PackQueuedRPCsForActor(TArray<TSharedRef<FPendingRPCParams>>* RPCList, AActor* Actor)
+{
+	RPCsOnEntityCreation QueuedRPCs;
+	check((RPCList != nullptr) && (Actor != nullptr));
+	if ((RPCList == nullptr) || (Actor == nullptr))
+	{
+		return QueuedRPCs;
+	}
+
+	UClass* Class = Actor->GetClass();
+	const FClassInfo& Info = ClassInfoManager->GetOrCreateClassInfoByClass(Class);
```

Comment:
I think you can delete the line above and just do it inline.
```suggestion
	const FClassInfo& Info = ClassInfoManager->GetOrCreateClassInfoByClass(Actor->GetClass());
```